web-components/src/components/modal/Modal.ts

```diff
@@ -163,7 +163,7 @@ export class Modal extends FocusTrapMixin(LitElement) {
     this.deactivateFocusTrap!();
 
     setTimeout(() => {
-      this.closeModal();
```

Comment:
This appears to be the only location where `private closeModal()` is used, so the function can be deleted from the file as well from line 112

Reply:
Agree! Deleted.